openedx/core/djangoapps/user_api/accounts/tests/test_views.py

```diff
@@ -244,7 +248,7 @@ def test_get_account_different_user_visible(self):
         """
         self.different_client.login(username=self.different_user.username, password=self.test_password)
         self.create_mock_profile(self.user)
-        with self.assertNumQueries(11):
+        with self.assertNumQueries(14):
```

Comment:
Because there are several `ConfigurationModels` that were being cached between tests before, and now are being read on every test.